shuup_workbench/settings/base_settings.py

```diff
@@ -198,32 +196,6 @@
 
 SHUUP_PRICING_MODULE = "customer_group_pricing"
 
-REST_FRAMEWORK = {
```

Comment:
These shouldn't be removed IMO since we going to install those exact same APIs from the new repositories.

Reply:
you're right